content/en/platform/corda/5.1/application-networks/managing/endpoints/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+date: '2023-09-21'
+title: "Member Endpoints"
+project: corda
+version: 'Corda 5.1'
+menu:
+  corda51:
+    identifier: corda51-networks-member-endpoints
+    parent: corda51-networks-manage
+    weight: 3000
+section_menu: corda51
+---
+# Member Endpoints
+
+If a member needs to make changes to their endpoint information, for example, changing the URL of the endpoint or adding a new endpoint, the member must re-register with the updated endpoint information. To minimize communication disruption, the member should re-register with both the old and new information, before removing the old information, as follows:
+
+1. Ensure that the cluster can accept connections on the new endpoint.
+   In a production environment that exposes the Corda P2P Gateway service via a load balancer, this may require updating the load balancer configuration with the new endpoint.
+2. Add the new endpoint information to the cluster's [P2P Gateway configuration]({{< relref "../../../deploying-operating/config/fields/p2p-gateway.md">}}) so that it listens on both the old and new endpoints. For example to configure two endpoints with different ports, using Bash with Curl or PowerShell:
+   {{< tabs >}}
+   {{% tab name="Bash"%}}
+   ```shell
+   export NEW_CONFIG='{
+   "config": {
+     "serversConfiguration":[{"hostAddress":"'$P2P_GATEWAY_HOST'","hostPort":8080,"urlPath":"/"}, {"hostAddress":"'$P2P_GATEWAY_HOST'","hostPort":8081,"urlPath":"/"}],"sslConfig":{"revocationCheck":{"mode":"OFF"} }
```

Comment:
We can remove `"sslConfig":{"revocationCheck":{"mode":"OFF"} }` to keep it generic. Also in other places below.

That field may have been present in the dev docs since we almost always have revocation disabled during internal use (and submitting a new config overwrites the previous version)